src/app/regular-giving/regular-giving.component.html

```diff
@@ -1,11 +1,12 @@
 <main class="b-container">
-  @if(!campaign.isRegularGiving) {
-      <div>
-          <p class="error">
-            Sorry, campaign {{ campaign.title }} is not a regular giving campaign.</p>
-          <p>To make a one-off donation to {{ campaign.charity.name }} please visit the regular <a href="/campaign/{{campaign.id}}">campaign page</a>.
-          </p>
-      </div>
+  @if (!campaign.isRegularGiving) {
+    <div>
+      <p class="error">
+        Sorry, campaign {{ campaign.title }} is not a regular giving campaign.</p>
+      <p>To make a one-off donation to {{ campaign.charity.name }} please visit the regular <a
```

Comment:
```suggestion
      <p>To make a one-off donation to {{ campaign.charity.name }} please visit the <a
```

Maybe best to avoid using 'regular' two different ways in the same page?

Reply:
Yes that's a good catch, thanks.